### ISSUE TITLE
NewPipe v0.27.1 is out 

### ISSUE THREAD

**Issue by danielwerg** [polymorphicshade/Tubular#108]
NewPipe v0.27.1 is out which includes fix for https://github.com/TeamNewPipe/NewPipe/issues/11255

#107 #106 #105 #101 #100 #98 #97

https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.27.1

**Comment by einsnulleins** [comment]
Before merging with 27.1, they have added new errors and the app is now entirely unusable for me; Sticking with the debug build from the newpipeextractor repo for now.
If they don't release a hotfix or >27.2, please see if you can fix this error in your fork.
Appreciate you :)

```
## Exception
* __User Action:__ requested kiosk
* __Request:__ Start loading: https://www.youtube.com/feed/trending
* __Content Country:__ DE
* __Content Language:__ en-DE
* __App Language:__ en_DE_#u-fw-mon-mu-celsius
* __Service:__ YouTube
* __Version:__ 0.27.1
* __OS:__ Linux Android 14 - 34
<details><summary><b>Crash log </b></summary><p>

java.net.UnknownHostException: Unable to resolve host "www.youtube.com": No address associated with hostname
	at java.net.Inet6AddressImpl.lookupHostByName(Inet6AddressImpl.java:124)
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Inet6AddressImpl.java:103)
	at java.net.InetAddress.getAllByName(InetAddress.java:1152)
	at okhttp3.Dns$Companion$DnsSystem.lookup(Dns.kt:49)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.kt:169)
	at okhttp3.internal.connection.RouteSelector.nextProxy(RouteSelector.kt:131)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.kt:73)
	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:205)
	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:106)
	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:74)
	at okhttp3.internal.connection.RealCall.initExchange$okhttp(RealCall.kt:255)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:32)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
	at okhttp3.internal.connection.RealCall.execute(RealCall.kt:154)
	at org.schabi.newpipe.DownloaderImpl.execute(DownloaderImpl.java:163)
	at org.schabi.newpipe.extractor.downloader.Downloader.get(Downloader.java:77)
	at org.schabi.newpipe.extractor.downloader.Downloader.get(Downloader.java:59)
	at org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.extractClientVersionFromHtmlSearchResultsPage(YoutubeParsingHelper.java:609)
	at org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.getClientVersion(YoutubeParsingHelper.java:681)
	at org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.prepareDesktopJsonBuilder(YoutubeParsingHelper.java:1173)
	at org.schabi.newpipe.extractor.services.youtube.YoutubeParsingHelper.prepareDesktopJsonBuilder(YoutubeParsingHelper.java:1157)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeTrendingExtractor.onFetchPage(YoutubeTrendingExtractor.java:66)
	at org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:60)
	at org.schabi.newpipe.extractor.kiosk.KioskInfo.getInfo(KioskInfo.java:53)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getKioskInfo$12(ExtractorHelper.java:197)
	at org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$Lfuzmkh5x8XukxOTFQPo4QLLzDQ(ExtractorHelper.java:0)
	at org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda7.call(R8$$SyntheticClass:0)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5377)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:16144)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5377)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:644)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:307)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)

</details>
<hr>
```

**Comment by rjdg14** [comment]
Are the intermittent 403 errors that have occured for the past few weeks in the middle of random videos, which 0.27.1 does not seem to fix, related to the change YouTube made that broke playback completely the other day (which it does fix), or are they two unrelated issues?

Until the periodic 403 errors in the middle of videos are fixed, the best way to ensure you'll be able to watch a video flawlessly seems to be to skip to a point past 0:59 seconds, and if the video continues playing normally then it shouldn't have any playback issues for the duration of the NewPipe session meaning you can rewind back to the start. If it results in the error then closing/reopening the app and selecting the video from your history should allow it to play normally.

**Comment by thecommitteduser** [comment]
Please push update to F-Droid repository please.

**Comment by shane-kerr** [comment]
If you use F-Droid and want the updated version you can add the NewPipe upstream repository:

https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/

You'll have to uninstall the F-Droid version due to incompatible signatures, and re-install (I lost all of my bookmarked playlist and downloads don't show up in NewPipe, although downloads are not deleted - so consider whether you would rather wait).

**Comment by thecommitteduser** [comment]
> If you use F-Droid and want the updated version you can add the NewPipe upstream repository:
> 
> https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
> 
> You'll have to uninstall the F-Droid version due to incompatible signatures, and re-install (I lost all of my bookmarked playlist and downloads don't show up in NewPipe, although downloads are not deleted - so consider whether you would rather wait).

Why would I add another repository, the purpose of F-Droid is to use a central trusted repository, adding another repository defeats the whole purpose of F-Droid to begin with.

**Comment by iointerrupt** [comment]
> Why would I add another repository, the purpose of F-Droid is to use a central trusted repository, adding another repository defeats the whole purpose of F-Droid to begin with.

You cannot simply "add" the apk to f-droid repo.  F-droid follows the standard CI/CD process where it will compile the app from source as well as all dependencies from the sources as well. This is to prevent malicious app devs from attempting to modify dependencies with malicious code and make their app code look "clean".

Since the process requires f-droid to compile every version of every app using their build servers, and f-droid is a community project funded mostly by donations, their resources are limited and compiling could take couple of days as there might be other apps in the f-droid pipeline being compiled and others waiting in the queue (first come first serve).

By adding the NewPipe repo you are telling f-droid that you trust their repo.  This way if NewPipe needs to make a hotfix it will be available quicker via f-droid.

Since NewPipe is sort of a "reactive" app where they cannot for see changes made by the YouTube team and have only fix after the fact, it makes sense to add the NewPipe repo to f-droid repo.

**Comment by thecommitteduser** [comment]
> > Why would I add another repository, the purpose of F-Droid is to use a central trusted repository, adding another repository defeats the whole purpose of F-Droid to begin with.
> 
> You cannot simply "add" the apk to f-droid repo. F-droid follows the standard CI/CD process where it will compile the app from source as well as all dependencies from the sources as well. This is to prevent malicious app devs from attempting to modify dependencies with malicious code and make their app code look "clean".
> 
> Since the process requires f-droid to compile every version of every app using their build servers, and f-droid is a community project funded mostly by donations, their resources are limited and compiling could take couple of days as there might be other apps in the f-droid pipeline being compiled and others waiting in the queue (first come first serve).
> 
> By adding the NewPipe repo you are telling f-droid that you trust their repo. This way if NewPipe needs to make a hotfix it will be available quicker via f-droid.
> 
> Since NewPipe is sort of a "reactive" app where they cannot for see changes made by the YouTube team and have only fix after the fact, it makes sense to add the NewPipe repo to f-droid repo.

I understand rhe benefits of using a direct repository (there is no right or wrong answer to this and everyone has a different preference) however if I were to do this for every app I won't know which will be a problematic repository in the future, there is a reason people use the Play Store over downloading APKs, and the same goes for F-Droid. There's no issue in waiting for a week for F-Droid to approve it, there's back up methods to use YouTube, however I've seen in the past F-Droid repo not have an update for 8 months since the last release when there's been over 5 to 10 releases done on the newpipe repo. Might as well decommission the F-Droid option on the website all together if it'll take that long for an update to be pushed.

There is a reason why my comment about F-Droid has half as many likes as the original post and that's because people do want a reliable upgrade schedule on F-Droid as that is their preferred solution.

**Comment by iointerrupt** [comment]
> I understand rhe benefits of using a direct repository (there is no right or wrong answer to this and everyone has a different preference) however if I were to do this for every app I won't know which will be a problematic repository in the future, there is a reason people use the Play Store over downloading APKs, and the same goes for F-Droid. There's no issue in waiting for a week for F-Droid to approve it, there's back up methods to use YouTube, however I've seen in the past F-Droid repo not have an update for 8 months since the last release when there's been over 5 to 10 releases done on the newpipe repo. Might as well decommission the F-Droid option on the website all together if it'll take that long for an update to be pushed.
> 
> There is a reason why my comment about F-Droid has half as many likes as the original post and that's because people do want a reliable upgrade schedule on F-Droid as that is their preferred solution.

I agree that a standard centralized library its best case for for most apps but how NewPipe or other similar apps/scripts like it (yt-dlp, youtube-dl) are slightly different.  Most apps might have a bug or two that might need to get patched because they werent properly tested during unit/regression testing.  If you are lucky then you can delete the current version and download an older version and wait for the author to fix and use that version until f-droid the fresh version appears on f-droid.

This is not how NewPipe works.  You cannot simply download an older version of NewPipe and expect things to work. Because the app deals with screen/web scraping, they are at the mercy of Google/Youtube to maintain consistency.  Because of this, if NewPipe is not working for one user, it is likely impacting all NewPipe users.  A week of downtime for the app might be too long, and YouTube app itself might not be an option especially for those Android users who are de-googled (some of my android devices are de-googled).

Its is a matter trust.  I trust F-Droid.  I have also used NewPipe long enough to realize that the NewPipe codebase is trustworthy and its managed by trustworthy contributors.  The F-Droid app allows you (the user) to extend that trust not just to its own repo, but any other repo the end user sees fit.

**Comment by thecommitteduser** [comment]
> > I understand rhe benefits of using a direct repository (there is no right or wrong answer to this and everyone has a different preference) however if I were to do this for every app I won't know which will be a problematic repository in the future, there is a reason people use the Play Store over downloading APKs, and the same goes for F-Droid. There's no issue in waiting for a week for F-Droid to approve it, there's back up methods to use YouTube, however I've seen in the past F-Droid repo not have an update for 8 months since the last release when there's been over 5 to 10 releases done on the newpipe repo. Might as well decommission the F-Droid option on the website all together if it'll take that long for an update to be pushed.
> > There is a reason why my comment about F-Droid has half as many likes as the original post and that's because people do want a reliable upgrade schedule on F-Droid as that is their preferred solution.
> 
> I agree that a standard centralized library its best case for most apps but how NewPipe or other similar apps/scripts like it (yt-dlp, youtube-dl) work are slightly different. Most apps might have a bug or two that might need to get patched because they weren't properly tested during unit/regression testing. If you are lucky then you can delete the current buggy version and download an older version and wait for the author to fix and use that version until fresh version appears on f-droid.
> 
> This is not how NewPipe works. You cannot simply download an older version of NewPipe and expect things to work. Because the app deals with screen/web scraping, they are at the mercy of Google/Youtube to maintain consistency. Because of this, if NewPipe is not working for one user, it is likely impacting all NewPipe users. A week of downtime for the app might be too long, and YouTube app itself might not be an option especially for those Android users who are de-googled (some of my android devices are de-googled).
> 
> Its is a matter trust. I trust the F-Droid repo. I have also used NewPipe long enough to realize that the NewPipe codebase is trustworthy and its managed by trustworthy contributors. The F-Droid app allows you (the user) to extend that trust not just to its own repo, but any other repo the end user sees fit.

You're stating the minority of cases (de googling, the average user doesn't even know what degoogling is) and you can use YouTube on any Web browser. I'm not advocating a replacement of the newpipe repository (if people want instant fixes that option will always be there), I'm advocating for submission to F-Droid straight away and letting them do their approval process, I haven't experienced that with my experience (I.e. Taking 8 months to publish to F-Droid after several releases last year, and a new release finally came out once a ticket was made suggesting the F-Droid repo to be updated.

If you want more contributors to a project you need to appeal to the masses (and the masses want F-Droid), and when popularity builds more devs will hear about the project and want to contribute. In turn this also increases the amount of people contributing to F-Droid too as more people will hear about new-pipe. It honestly reminds me of the endless loop of people thinking their Linux distro is how it should be done, when all they need to do is understand want the average user wants.

**Comment by iointerrupt** [comment]
> You're stating the minority of cases (de googling, the average user doesn't even know what degoogling is) and you can use YouTube on any Web browser. I'm not advocating a replacement of the newpipe repository (if people want instant fixes that option will always be there), I'm advocating for submission to F-Droid straight away and letting them do their approval process, I haven't experienced that with my experience (I.e. Taking 8 months to publish to F-Droid after several releases last year, and a new release finally came out once a ticket was made suggesting the F-Droid repo to be updated.
> 
> If you want more contributors to a project you need to appeal to the masses (and the masses want F-Droid), and when popularity builds more devs will hear about the project and want to contribute. In turn this also increases the amount of people contributing to F-Droid too as more people will hear about new-pipe. It honestly reminds me of the endless loop of people thinking their Linux distro is how it should be done, when all they need to do is understand want the average user wants.
> 
> Telling people what they should want isn't a good approach. If the majority of the people want something you give it to them. You don't go telling them that's the wrong approach and they should use a repo they don't want to use. They heard the reasons why a direct repo is beneficial and they still prefer F-droid. The majority of users aren't coders to evaluate whether they should trust an independent repo and many don't have to time to evaluate every single project codebase they use.

While "de-googling" might be a minority case, I doubt the the the masses of Android users are installing F-Droid.  First of all, you are downloading an APK that is not signed by Google/Amazon/Samsung/etc PlayStore.  As soon as you  attempt to launch it, Google Play Services or phone tablet OEM security warns you that your are about to install a package than can be harmful.  Furthermore, depending on the manufacturer, you may have to enable this capability in some option buried deep in settings.  Even further,  once installed, you will now have to grant F-Droid the app permission to install apps from "unknown sources" from within itself, which requires another layer prompts to ensure the users understand what they are about to do.  All of this is to deter the masses of android user from installing apps outside the security sandbox.  All of this is actually for good reason.  You should have some understanding of the implications of what you are about to do, and if you don't, your best option is to click cancel/no. 

Youtube does offer users Youtube Premium service and they advertise it well enough. This service also ensures, the average android user does not need to worry about what a repository is or the need to trust them for that matter. My aging parents both have YouTube. They don't mind the advertisements.  But the moment they complain, I will immediately them to YouTube Premium.  Because I understand how a day or two downtime could impact them.  Because its expected with a project like NewPipe and that it can fail F-Droids checks which could potentially delay it further. I can do with the workaround solutions, because I am far more tech savvy than they are.  Looking for workarounds is not something they not need to worry about because they are your average android users.

**Comment by thecommitteduser** [comment]
> > You're stating the minority of cases (de googling, the average user doesn't even know what degoogling is) and you can use YouTube on any Web browser. I'm not advocating a replacement of the newpipe repository (if people want instant fixes that option will always be there), I'm advocating for submission to F-Droid straight away and letting them do their approval process, I haven't experienced that with my experience (I.e. Taking 8 months to publish to F-Droid after several releases last year, and a new release finally came out once a ticket was made suggesting the F-Droid repo to be updated.
> > If you want more contributors to a project you need to appeal to the masses (and the masses want F-Droid), and when popularity builds more devs will hear about the project and want to contribute. In turn this also increases the amount of people contributing to F-Droid too as more people will hear about new-pipe. It honestly reminds me of the endless loop of people thinking their Linux distro is how it should be done, when all they need to do is understand want the average user wants.
> > Telling people what they should want isn't a good approach. If the majority of the people want something you give it to them. You don't go telling them that's the wrong approach and they should use a repo they don't want to use. They heard the reasons why a direct repo is beneficial and they still prefer F-droid. The majority of users aren't coders to evaluate whether they should trust an independent repo and many don't have to time to evaluate every single project codebase they use.
> 
> While "de-googling" might be a minority case, I doubt the the the masses of Android users are installing F-Droid. First of all, you are downloading an APK that is not signed by Google/Amazon/Samsung/etc PlayStore. As soon as you attempt to launch it, Google Play Services or phone tablet OEM security warns you that your are about to install a package than can be harmful. Furthermore, depending on the manufacturer, you may have to enable this capability in some option buried deep in settings. Even further, once installed, you will now have to grant F-Droid the app permission to install apps from "unknown sources" from within itself, which requires another layer prompts to ensure the users understand what they are about to do. All of this is to deter the masses of android user from installing apps outside the security sandbox. All of this is actually for good reason. You should have some understanding of the implications of what you are about to do, and if you don't, your best option is to click cancel/no.
> 
> Youtube does offer users Youtube Premium service and they advertise it well enough. This service also ensures, the average android user does not need to worry about what a repository is or the need to trust them for that matter. My aging parents both have YouTube. They don't mind the advertisements. But the moment they complain, I will immediately subscribe them to YouTube Premium. Because I understand how a day or two downtime could impact them. Because its expected with a project like NewPipe and that it can fail F-Droids checks which could potentially delay it further. I can do with the workaround solutions, because I am far more tech savvy than they are. Looking for workarounds is not something they not need to worry about because they are your average android users.

It's not one extreme or another, F-Droid is the best middle ground for many people. If it weren't the majority of people using newpipe then my first comment wouldn't have so many thumbs up. Again, if it isn't believed as a good solution then why is it offered on F-Droid in the first place.

Anyways, I'm advocating for many options (and done properly if it's offered as an option) to appeal to everyone and you're stating no, my way is the right way and if they don't like that method go somewhere else which isn't good community building. I don't think there is much else to be said.

**Comment by iointerrupt** [comment]
> It's not one extreme or another, F-Droid is the best middle ground for many people. If it weren't the majority of people using newpipe then my first comment wouldn't have so many thumbs up. Again, if it isn't believed as a good solution then why is it offered on F-Droid in the first place.
> 
> Anyways, I'm advocating for many options (and done properly if it's offered as an option) to appeal to everyone and you're stating no, my way is the right way and if they don't like that method go somewhere else which isn't good community building. I don't think there is much else to be said.

Can you please reference any comment where I said it anything along the lines of "my way, and if you like that method go somewhere else"? How is being able to add additional repositories to the F-Droid app not advocating for additional options? F-Droid docs tell you how to do it here: https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/

If you want to learn a bit more about the f-droid build process, you can do so here:
https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client.

I also want NewPipe to get released on F-Droids official repo and have given your comment above a thumbs up.

**Comment by thecommitteduser** [comment]
> > It's not one extreme or another, F-Droid is the best middle ground for many people. If it weren't the majority of people using newpipe then my first comment wouldn't have so many thumbs up. Again, if it isn't believed as a good solution then why is it offered on F-Droid in the first place.
> > Anyways, I'm advocating for many options (and done properly if it's offered as an option) to appeal to everyone and you're stating no, my way is the right way and if they don't like that method go somewhere else which isn't good community building. I don't think there is much else to be said.
> 
> Can you please reference any comment where I said it anything along the lines of "my way, and if you like that method go somewhere else"? How is being able to add additional repositories to the F-Droid app not advocating for additional options? F-Droid docs tell you how to do it here: https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/
> 
> If you want to learn a bit more about the f-droid build process, you can do so here: https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client.
> 
> I also want NewPipe to get released on F-Droids official repo and have given your comment above a thumbs up.

My apologies then. My misunderstanding. I believe Everyone is aware another repo can be used, even newpipe prompts there are new updates by downloading an apk. There just seems to be a reluctance to push releases through F-Droid (I don't think 8 months is because of F-Droids approval process) and the stance of benefits you provided to me implies that people should just use the other methods when all I initially suggested was to upload to F-Droid please. However still, it's my misunderstanding and my fault and I'm sorry for assuming that was your position.

**Comment by iointerrupt** [comment]
> My apologies then. My misunderstanding. I believe Everyone is aware another repo can be used, even newpipe prompts there are new updates by downloading an apk. There just seems to be a reluctance to push releases through F-Droid (I don't think 8 months is because of F-Droids approval process) and the stance of benefits you provided to me implies that people should just use the other methods when all I initially suggested was to upload to F-Droid please. However still, it's my misunderstanding and my fault and I'm sorry for assuming that was your position.

No worries man.  8 months sound a bit extreme for either end of the link.  Hopefully both NewPipe and F-Droid get continued support from us to streamline and move the process along faster.

Now that NewPipe is working again, I will be enjoying some MST3K shorts on it before calling it a night.

**Comment by NewMacPro** [comment]
V0.27.1 can't play again
org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: "Sign in to confirm that you're not a bot"
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.checkPlayabilityStatus(YoutubeStreamExtractor.java:948)
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.onFetchPage(YoutubeStreamExtractor.java:844)
	at org.schabi.newpipe.extractor.Extractor.fetchPage(Extractor.java:60)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:77)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:72)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:118)

**Comment by Alexia99-lab** [comment]
Latest version 0.27.1 still same network error
![Screenshot_20240725-044210](https://github.com/user-attachments/assets/c6ca421c-ae52-4047-a045-0275d51d3458)

**Comment by alexshpilkin** [comment]
@Alexia99-lab This is a change on YouTube’s side that seems to have been deployed literally today. See the comment in the upstream bug tracker at https://github.com/TeamNewPipe/NewPipe/issues/11255#issuecomment-2248098119. So As here is not that bug tracker, we

**Comment by alexshpilkin** [comment]
@Alexia99-lab This is a change on YouTube’s side that seems to have been deployed literally today. See the comment in the upstream bug tracker at https://github.com/TeamNewPipe/NewPipe/issues/11255#issuecomment-2248098119. The peoblem has already been fixed (as has the different bug with crashes around the 1-minute mark, see https://github.com/TeamNewPipe/NewPipe/issues/11191), so the only thing we can do here in this fork is to wait for 0.27.2 or whatever it’s going to end up being called, then ask the maintainer for a merge.

(Adventurous and knowledgeable folks could try and integrate the changes themselves, build that and install it on their Android devices. But that sounds like a lot of fairly drudgy work—hats off to the @polymorphicshade—for an issue that’s likely to see a proper solution within a week.)

**Comment by alexshpilkin** [comment]
@polymorphicshade In the meantime, upstream has released [0.27.2](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.27.2).
